keyboards/jcpm2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,154 @@
+    ),
+};
+
+#ifdef ENCODER_MAP_ENABLE
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
+    [_LAYER0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) }, // default
+    [_LAYER0_MOD] = { ENCODER_CCW_CW(_______, _______) }, // default MOD
+    [_LAYER1] = { ENCODER_CCW_CW(KC_LEFT, KC_RIGHT) }, // FCPX
+    [_LAYER2] = { ENCODER_CCW_CW(KC_R, S(KC_R)) }, // KICAD
+    [_LAYER3] = { ENCODER_CCW_CW(_______, _______) }, // UNUSED
+};
+#endif
+
+#ifdef OLED_ENABLE
+// Function to display the current layer and information on the OLED
+bool oled_task_user(void) {
+    switch (get_highest_layer(layer_state | default_layer_state)) {
+        case _LAYER0:
+            oled_write_ln_P(PSTR("(VOL- VOL+)  LAYER0"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("MUTE      SLOW FAST"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("LAYR RSET 10BK 10FW"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("MREV MPLA MFWD +OPT"), false);
+            break;
+        case _LAYER0_MOD:
+            oled_write_ln_P(PSTR("____ ____      0-MOD"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("____      BRI+ ANIM"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("____ ____ BRI- TOGG"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("EMOJ ____ ____ -OPT"), false);
+            break;
+        case _LAYER1:
+            oled_write_ln_P(PSTR("(FRA- FRA+)    FCPX"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("XXXX      BRAK ZOM+"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("LAYR RSET KFRA ZOM-"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("BACK STOP FORW DELT"), false);
+            break;
+        case _LAYER2:
+            oled_write_ln_P(PSTR("(RO L RO R)   KICAD"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("EDIT      ESCP MOVE"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("LAYR RSET UNDO TRAK"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("VIA  DRAG SALL DELT"), false);
+            break;
+        case _LAYER3: //not used
+            oled_write_ln_P(PSTR("(RO L RO R)   MIDI"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("EDIT      B3   C4"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("LAYR RSET G3   A3"), false);
+            oled_write_ln_P(PSTR(""), false);
+            oled_write_ln_P(PSTR("C3   D3   E3   F3"), false);
+            break;
+    }
+    return false;
+}
+#endif
+
+void rgb_matrix_indicators_user(void) {
```

Comment:
Sorry, bad copy paste...
```suggestion
bool rgb_matrix_indicators_user(void) {
```